Livecheckables/telnet.rb

```diff
@@ -0,0 +1,6 @@
+class Telnet
+  livecheck do
+    url "https://opensource.apple.com/source/remote_cmds/"
+    regex(/href=.*?remote_cmds-v?(\d+(?:\.\d+)*)/i)
```

Comment:
```suggestion
    url "https://opensource.apple.com/tarballs/remote_cmds/"
    regex(/href=.*?remote_cmds-v?(\d+(?:\.\d+)*)\.t/i)
```

The formula uses an archive from https://opensource.apple.com/tarballs/remote_cmds/, so it seems appropriate to check that index page for new versions. Was there a reason to check the "source" directories instead?